packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2098,6 +2150,32 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   @protected
   bool get alwaysNeedsCompositing => false;
 
+  late bool _wasRepaintBoundary;
+
+  /// Update the composited layer owned by this render object.
+  ///
+  /// This method is called by the framework when [isRepaintBoundary] is true.
+  ///
+  /// If [oldLayer] is `null`, this method should return a new [OffsetLayer]
```

Comment:
The "should" in this and the next sentence should really be "must"s, no? There is no other option.

Reply:
done